docs/admin/access.rst

```diff
@@ -1,333 +1,433 @@
 
-.. versionadded:: 3.11
+  2. Add a dedicated group for `Czech` translators.
 
-* Resend the e-mail for user invitations (invalidating any previously sent invitation)
+     .. list-table:: Group `Czech translators`
+         :stub-columns: 1
 
-User management is available in the :guilabel:`Manage` menu of any project:
+         * - Roles
+           - `Power users`
+         * - Project selection
+           - `All public projects`
+         * - Language selection
+           - `As defined`
+         * - Languages
+           - `Czech`
 
-.. image:: /images/manage-users.png
+..
 
-.. seealso::
+  3. Add to the group some users you wish to give the permissions to.
+
+As you can see, permissions management this way is a powerful tool.
+Unfortunately, on the down side it can be quite a tedious job. And you can't
+delegate it to another user without granting superuser permissions.
 
-   :ref:`acl`
+.. _auth-model:
 
-.. _groups:
+Users, roles, groups and permissions
+++++++++++++++++++++++++++++++++++++
 
-Predefined groups
-+++++++++++++++++
+The authentication models consist of several objects:
 
-Weblate comes with a predefined set of groups for a project, wherefrom you can assign
-users.
+`Permission`
+    Individual permissions defined by Weblate. Permissions can not be
+    assigned to users. This can only be done through assignment of roles.
+`Role`
+    A Role defines a set of permissions. This allows reuse of these sets in
+    several places, making the administration easier.
+`User`
+    Users can belong to several groups.
+`Group`
+    Groups connect roles, users and authentication objects (projects,
+    languages and component lists).
 
+.. graphviz::
 
-.. describe:: Translate
+    graph auth {
 
-    Can translate the project, and upload translations made offline.
+        "User" -- "Group";
+        "Group" -- "Role";
+        "Role" -- "Permission";
+        "Group" -- "Project";
+        "Group" -- "Language";
+        "Group" -- "Components";
+        "Group" -- "Component list";
+    }
 
-.. describe:: Sources
+.. note::
 
-    Can edit source strings in :ref:`monolingual` and source string info.
+  A group can have no roles assigned to it, in that case access to browse the
+  project is assumed (see bellow).
 
-.. describe:: Languages
+Access for browse to a project
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    Can manage translated languages (add or remove translations).
+A user has to be a member of a group linked to the project, or any component
+inside that project. Having membership is enough, no specific permissions are
+needed to browse the project (this is used in the default `Viewers` group, see
+:ref:`default-groups`).
 
-.. describe:: Glossary
+Access for browse to a component
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    Can manage glossary (add or remove entries, or upload).
+A user can access unrestricted components once able to access the components'
+project (and will have all the permissions he or she was granted for the
```

Comment:
```suggestion
project (and will have all the permissions the user was granted for the
```
Isn't it by way of being in a group assigned to the project? (I don't know how access control works).

Reply:
> Isn't it by way of being in a group assigned to the project? (I don't know how access control works.)

Yes, it is, but I believe to elaborate it here is a bit excessive...